src/modules/esp32/PaxcounterModule.cpp

```diff
@@ -31,9 +49,12 @@ bool PaxcounterModule::sendInfo(NodeNum dest)
     meshtastic_MeshPacket *p = allocDataProtobuf(pl);
     p->to = dest;
     p->decoded.want_response = false;
-    p->priority = meshtastic_MeshPacket_Priority_MIN;
+    p->priority = meshtastic_MeshPacket_Priority_DEFAULT;
```

Comment:
Please leave this as min, regular message traffic should have priority.

Reply:
ok, no problem.
Just a question: When would BACKGROUND be appropriate?

Reply:
Pretty well explained in https://github.com/meshtastic/protobufs/blob/00332412b238fe559175a6e83fdf8d31fa5e209a/meshtastic/mesh.proto#L819-L878 ... I think BACKGROUND may be more appropriate than MIN here, and in other places that use it. (Telemetry and so on...) For now i'd stick to min cause the other plugins use min too. Let's address this in a different PR.